packages/gitbeaker-cli/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "@gitbeaker/cli",
+  "description": "Full NodeJS CLI implementation of the GitLab API.",
+  "version": "15.0.0",
+  "author": {
+    "name": "Justin Dalrymple"
+  },
+  "bin": {
+    "gitbeaker": "dist/index.js"
+  },
+  "bugs": {
+    "url": "https://github.com/jdalrymple/gitbeaker/issues"
+  },
+  "dependencies": {
+    "@gitbeaker/core": "15.0.0",
```

Comment:
This should be pulled in from `@gitbeaker/node`?

Reply:
I need to import the map generated in the core module :( The good thing is that the rollup tree shaking will only pull the json file and not the whole package

Reply:
You should still have access to the dep because `@gitbeaker/node` depends on it? 🤔

Reply:
Its its not included in the dependencies, I cant import it in the code though... Unless I'm missing something.

Reply:
Just waiting on thoughts for this^

Reply:
might be different with lerna so seems fine 😅